Sources/ProjectSpec/SpecValidation.swift

```diff
@@ -36,7 +36,7 @@ extension ProjectSpec {
             if !(basePath + configFile).exists {
                 errors.append(.invalidConfigFile(configFile: configFile, config: config))
             }
-            if getConfig(config) == nil {
+            if !options.disabledValidations.contains(.missingConfigs) && getConfig(config) == nil {
```

Comment:
I guess we can also add this check to https://github.com/yonaskolb/XcodeGen/blob/master/Sources/ProjectSpec/SpecValidation.swift#L20